core/loadpoint.go

```diff
@@ -817,6 +817,10 @@ func (lp *LoadPoint) setActiveVehicle(vehicle api.Vehicle) {
 	}
 	lp.log.INFO.Printf("vehicle updated: %s -> %s", from, to)
 
+	// reset minSoC and targetSoC before change
+	lp.setMinSoC(0)
+	lp.setTargetSoC(0)
```

Comment:
0 bedeutet in der Regelung "Kein Limit"?

Reply:
Genau- sonst wäre der Wert ja Unsinn.